docs/utilities/tags.md

```diff
@@ -21,17 +21,45 @@ Using Tags In Code
 ------------------
 Block, Item, and Fluid tags are automatically sent from the server to any remote clients on login and reload. Function tags are not synced.
 
+Tags can be accessed in code through the `Tags.Items` and `Tags.Blocks` classes for all the Vanilla Minecraft tags, and through `ResourceLocation`s for ForgeTags.
```

Comment:
This isn't true, in fact it's backwards. `Tags` (`net.minecraftforge.common.tags.Tags`) holds the *forge* tags. Vanilla tags come from `ItemTags`, `BlockTags`, etc. And you would never access tags "through `ResourceLocation`s".